apps/kg-explorer/src/EMBEDDING.md

```diff
@@ -8,7 +8,7 @@ The API endpoint and mirror url can be set as in this example:
   <head>
     <meta charset="utf-8" />
     <title>HRA Knowledge Graph Explorer</title>
-    <base href="/" />
+    <base href="https://cdn.humanatlas.io/digital-objects" />
```

Comment:
That's not what I meant @edlu77 will stop by in a minute, because that definitely doesn't work.